src/server/pfs/server/commit_store.go

```diff
@@ -228,6 +228,11 @@ func setTotal(tx *pachsql.Tx, tr track.Tracker, commit *pfs.Commit, id fileset.I
 	if err := tr.CreateTx(tx, oid, pointsTo, track.NoTTL); err != nil {
 		return errors.EnsureStack(err)
 	}
+	if exists, err := getRepoExists(tx, commit); err != nil {
+		return errors.EnsureStack(err)
+	} else if !exists {
+		return nil // repo has been removed - do not update commit totals
```

Comment:
We shouldn't silently exit here.  We should return some kind of error saying the Repo (or commit) doesn't exist.